mods/seasonal/moves.js

```diff
@@ -109,6 +109,52 @@ let BattleMovedex = {
 		type: "Electric",
 		zMovePower: 200,
 	},
+	// Andy
+	pilfer: {
+		accuracy: 100,
+		basePower: 70,
+		category: "Physical",
+		desc: "",
+		shortDesc: "",
+		id: "pilfer",
+		name: "Pilfer",
+		pp: 5,
+		priority: 1,
+		flags: {protect: 1, mirror: 1, contact: 1},
+		onPrepareHit: function (target, source) {
+			this.attrLastMove('[still]');
+		},
+		onTryHit: function (target, pokemon) {
+			let decision = this.willMove(target);
+			if (decision) {
+				let move = this.getMoveCopy(decision.move.id);
+				if (move.category === 'Status' && move.id !== 'mefirst') {
+					this.useMove(move, pokemon, pokemon);
+					this.attrLastMove('[still]');
+					this.add('-anim', pokemon, "Sucker Punch", target);
+					this.add('-anim', pokemon, "Night Slash", target);
```

Comment:
anim should be in onPrepareHit rather than here (play reg move anim after plifering anim)